main.go

```diff
@@ -24,6 +24,7 @@ var (
 	listenAddress = kingpin.Flag("listen", "Address for 9volt's API to listen on").Short('l').Default("0.0.0.0:8080").String()
 	etcdPrefix    = kingpin.Flag("etcd-prefix", "Prefix that 9volt's configuration is stored under in etcd").Short('p').Default("9volt").String()
 	etcdMembers   = kingpin.Flag("etcd-members", "List of etcd cluster members").Short('e').Required().Strings()
+	debugUI       = kingpin.Flag("debug-ui", "Debug the user interface locally").Short('u').Bool()
```

Comment:
Here's the flag. Me no like the short one I chose (`u`), but whatever, it was available. Ultimately, I don't even know if I like this as a flag. But for development purposes, I needed to find some way to allow running the website while making changes without compiling the statik resources. The `npm install` can sometimes be slow and the webpack for sure can be slow.